ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-commands.sh

```diff
@@ -4,24 +4,49 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
 # TODO: move to image
 curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
 
+EXPIRATION_DATE=$(date -d '4 hours' --iso=minutes --utc)
+TAGS="Key=expirationDate,Value=${EXPIRATION_DATE}"
+
 CONFIG="${SHARED_DIR}/install-config.yaml"
 PATCH="${SHARED_DIR}/install-config-blackholenetwork.yaml.patch"
 
-aws_region=$(/tmp/yq r "${CONFIG}" 'platform.aws.region')
+CLUSTER_NAME="$(/tmp/yq r "${CONFIG}" 'metadata.name')"
+REGION=$(/tmp/yq r "${CONFIG}" 'platform.aws.region')
+
+curl -L https://raw.githubusercontent.com/openshift/release/master/ci-operator/step-registry/ipi/conf/aws/blackholenetwork/blackhole_vpc_yaml.md -o /tmp/blackhole_vpc.yaml
```

Comment:
I think I would prefer to have the contents of this file directly in the script rather than having to curl in every job instance.